paper.md

```diff
@@ -0,0 +1,89 @@
+---
+title: 'ivadomed: A Medical Imaging Deep Learning Toolbox'
+tags:
+  - Deep Learning
+  - Medical Imaging
+  - Segmentation
+  - Open-source
+  - Pipeline
+authors:
+  - name: Charley Gros
+    orcid: 0000-0003-4318-0024
+    affiliation: 1
+  - name: Andreanne Lemay
+    orcid: XX
+    affiliation: 1
+  - name: Olivier Vincent
+    orcid: XX
+    affiliation: 1
+  - name: Lucas Rouhier
+    orcid: XX
+    affiliation: 1
+  - name: Anthime Bucquet
+    orcid: XX
+    affiliation: 1
+  - name: Joseph Paul Cohen
+    orcid: XX
+    affiliation: "2, 3"
+  - name: Julien Cohen-Adad
+    orcid: 0000-0003-3662-9532
+    affiliation: "1, 4"
+affiliations:
+ - name: NeuroPoly Lab, Institute of Biomedical Engineering, Polytechnique Montreal, Montreal, Canada
+   index: 1
+ - name: AIMI, Stanford University, Stanford, CA, USA
+   index: 2
+ - name: Mila, Quebec AI Institute, Montreal, QC, Canada
+   index: 3
+ - name:  Functional Neuroimaging Unit, CRIUGM, Université de Montréal, Montreal, QC, Canada
+   index: 4
+date: 3 November 2020
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`ivadomed` is an open-source Python package for designing, end-to-end training, and evaluating deep learning models applied to medical imaging data. The package includes APIs, command-line tools, documentation, and tutorials. `ivadomed` also includes pre-trained models such as spinal tumor segmentation and vertebral labeling. Original features of `ivadomed` include a data loader that can parse image and subject metadata for custom data splitting or extra information during training and evaluation. Any dataset following the [Brain Imaging Data Structure (BIDS)](https://bids.neuroimaging.io/) convention will be compatible with `ivadomed`. Beyond the traditional deep learning methods, `ivadomed` features cutting-edge architectures, such as FiLM [@perez2017film] and HeMis [@havaei2016hemis], as well as various uncertainty estimation methods (aleatoric and epistemic), and losses adapted to imbalanced classes and non-binary predictions. Example applications of `ivadomed` include MRI object detection, segmentation, and labeling of anatomical and pathological structures. `ivadomed`'s main project page is available at https://ivadomed.org.
+
+# Statement of need
+
+Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison-with-other-projects). `ivadomed` notably address unmet needs in terms of data management, readily-available uncertainty outputs, missing channels (in case of multi-channel training) and model comparison, to only name few of the original features.
```

Comment:
```suggestion
Deep learning applied to medical imaging presents many challenges: datasets are often not publicly-available, ground truth labels are difficult to produce, and needs are usually tailored to particular datasets and tasks [@kim_deep_2019]. There already exists a few deep learning frameworks for medical imaging, but [each have their pros & cons](https://ivadomed.org/en/latest/purpose.html#comparison-with-other-projects). `ivadomed` notably address unmet needs in terms of data management, readily-available uncertainty outputs, missing modalities (in case of multi-channel training) and model comparison, to only name few of the original features.
```